etcd/README.md

```diff
@@ -0,0 +1,73 @@
+# Etcd
+
+A Etcd storage driver using [`etcd-io/etcd`](https://github.com/etcd-io/etcd).
+
+### Table of Contents
+- [Signatures](#signatures)
+- [Installation](#installation)
+- [Examples](#examples)
+- [Config](#config)
+- [Default Config](#default-config)
+
+### Signatures
+```go
+func New(config ...Config) *Storage
+func (s *Storage) Get(key string) ([]byte, error)
+func (s *Storage) Set(key string, val []byte, exp time.Duration) error
+func (s *Storage) Delete(key string) error
+func (s *Storage) Reset() error
+func (s *Storage) Close() error
+func (s *Storage) Conn() *clientv3.Client
+```
+
+### Installation
+Memory is tested on the 2 last [Go versions](https://golang.org/dl/) with support for modules. So make sure to initialize one first if you didn't do that yet:
+```bash
+go mod init github.com/<user>/<repo>
+```
+And then install the memory implementation:
```

Comment:
Please replace it with your storage

Reply:
Fixed README.md for etcd on [this](https://github.com/gofiber/storage/pull/878/commits/9f8afc3ce948d09731247888e6cb8a22e0438f0b) commit